src/Control/CookieJar.php

```diff
@@ -152,7 +152,7 @@ public function forceExpiry($name, $path = null, $domain = null, $secure = false
      *
      * @param string $name The name of the cookie
      * @param string|array $value The value for the cookie to hold
-     * @param int $expiry The number of days until expiry
+     * @param int $expiry The time the cookie expires. 0 expires at the end of the session
```

Comment:
Not an externally facing API per se, but still helpful: Maybe to differentiate from the other `$expiry`'s, should we call this out as being a Unix timestamp? Also makes me wonder why the default `$expiry = 90` was ever defined in the parameter list below, since that'd be back in the 1970's 🤔

Reply:
I pulled that from the php docs. I also considered modifying the default to more congruent with the expected value, but am trying to keep the functional impact as little as possible

Reply:
Agreed on lowering impact. I'd imagine an `$expiry = 0` would have the same effect (unless your system's clock is >40yrs off) and would be less confusing, considering `90` looks like days whereas `0` implies expired/empty (which `90 seconds after epoch` effectively would be)..

Reply:
Yeah, `90` isn't a very good value. But if a user has subclassed `CookieJar` and is relying on this default, then I wouldn't want this changeset to break their code.